apps/assistants/views.py

```diff
@@ -140,6 +159,18 @@ def form_valid(self, form):
         return response
 
 
+class SyncEditingOpenAiAssistant(BaseOpenAiAssistantView, View):
+    permission_required = "assistants.change_openaiassistant"
+
+    def post(self, request, team_slug: str, pk: int):
+        assistant = get_object_or_404(OpenAiAssistant, team=request.team, pk=pk)
```

Comment:
Nit: doing `request.team` will fire a DB query. Rather check the team using the `team_slug`

Reply:
I forgot why I did this and made the change and remembered, team_slug is not in the correct format and causes and error when used instead. This is akso used in a few other views on the page so if you have and ideas lmk and I can fix them all

Reply:
This should be fine since we'll almost always have to load the team for a request. But if you do want to use `team_slug` you'll need to do it like this `team__slug=team_slug`